vm-setup/roles/common/tasks/main.yml

```diff
@@ -18,3 +18,20 @@
 - debug:
     var: vm_nodes
   when: generate_vm_nodes
+
+- set_fact:
```

Comment:
Can we add a name for this to track the task easily in jenkins logs?

Reply:
Now done, thanks!